draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -9545,10 +9590,40 @@ Content-Encoding: gzip
 </section>
 
 <section title="Header Field Registration" anchor="header.field.registration">
+
+<t>      
+   Please create a new registry as outlined in <xref
+   target="field.name.registry"/>.
+</t>
+
+<t>
+   After creating the registry, all entries in the Permanent and Provisional
+   Message Header Registries with the protocol 'http' are to be moved to it,
+   with the following changes applied:
+</t>
+
+<ol>
+   <li><t>The 'Applicable Protocol' field is to be omitted.</t></li>
+   <li><t>Entries with a status of 'standard', 'experimental', or
+   'informational' are to have a status of 'permanent'.</t></li>
+   <li><t>Entries with a status of 'deprecated' are to have a status of
+   'obsoleted'.</t></li>
```

Comment:
I'd expected to see a mention of 'obsoleted' in the field.name.registry section if it's going to be used. Should the "Provisional entries can be removed by the Expert(s)" permission instead on in addition say the Expert can mark entries 'obsoleted'?

Reply:
Any registry can have obsoleted entries; it comes with the territory (and the registration section is targeted at people registering new ones). 

For provisional, the point is to reclaim the entries if they aren't used. The experts *can* mark them obsoleted if they like; IANA instructions are much less prescriptive than you might think.